<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="ar_lbl">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>110</y>
       <width>251</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../UI_img/알람시계.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>40</y>
       <width>50</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>알</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>73</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>람</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>192</x>
       <y>40</y>
       <width>50</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>빠</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>73</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>르게  안끄</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>40</y>
       <width>50</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>노</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>280</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(170, 170, 255);
border-style:solid;
border-width:2px;
</string>
     </property>
     <property name="placeholderText">
      <string>  아이디를 입력해주세요</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>318</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(170, 170, 255);
border-style:solid;
border-width:2px;
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>  비밀번호를 입력해주세요</string>
     </property>
    </widget>
    <widget class="QLabel" name="error_lbl">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>364</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>아이디 또는 비밀번호를 입력해주세요.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_btn">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>382</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(110, 110, 255);
color: rgb(255, 255, 255);
border-width:0px;
border-style:solid;
</string>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>430</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>431</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(97, 97, 97);</string>
     </property>
     <property name="text">
      <string>자동 로그인</string>
     </property>
    </widget>
    <widget class="QLabel" name="join_lbl">
     <property name="geometry">
      <rect>
       <x>289</x>
       <y>431</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(97, 97, 97);</string>
     </property>
     <property name="text">
      <string>알빠노 회원가입</string>
     </property>
    </widget>
    <widget class="QLabel" name="ani_lbl">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>472</y>
       <width>211</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../UI_img/늦잠3.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>40</y>
       <width>50</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>알</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>73</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>람</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>191</x>
       <y>40</y>
       <width>50</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>빠</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>73</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>르게  안끄</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>294</x>
       <y>40</y>
       <width>50</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>노</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_line">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>280</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(170, 170, 255);
border-width:2px;
border-style:solid;
</string>
     </property>
     <property name="placeholderText">
      <string> 아이디를 입력해주세요</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pw_line">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>250</y>
       <width>280</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(170, 170, 255);
border-width:2px;
border-style:solid;
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string> 비밀번호를 입력해주세요</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pw_checkline">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>300</y>
       <width>280</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(170, 170, 255);
border-width:2px;
border-style:solid;
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string> 비밀번호를 확인해주세요</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>208</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>아이디              </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>260</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>비밀번호          </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>310</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>비밀번호 확인 </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>362</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>닉네임</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_line">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>355</y>
       <width>280</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(170, 170, 255);
border-width:2px;
border-style:solid;
</string>
     </property>
     <property name="placeholderText">
      <string> 닉네임을 입력해주세요</string>
     </property>
    </widget>
    <widget class="QPushButton" name="join_btn">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>425</y>
       <width>365</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(110, 110, 255);
color: rgb(255, 255, 255);
border-width:0px;
border-style:solid;</string>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_btn">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>475</y>
       <width>365</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(110, 110, 255);
color: rgb(255, 255, 255);
border-width:0px;
border-style:solid;</string>
     </property>
     <property name="text">
      <string>뒤로가기</string>
     </property>
    </widget>
    <widget class="QLabel" name="error_lbl_2">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>400</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>아이디가 이미 존재합니다.</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
